<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="test"
    errors="0" failures="1" hostname="" id="" package="coffee" skipped="0"
    tests="1" time="3.815" timestamp="Wed, 06 Apr 2016 18:24:09 GMT">
  
    <testcase name="it should display custom new tab" time="3.815" assertions="2">

      <failure message="Testing if element &lt;#message&gt; contains text: &quot;undefined&quot;.">  at Object.module.exports.it should display custom new tab (D:\Dev\newtab\tests\js\coffee\test.js:3:86)
  at Module.call (D:\Dev\newtab\node_modules\nightwatch\lib\runner\module.js:63:34)
  at D:\Dev\newtab\node_modules\nightwatch\lib\runner\testcase.js:97:29
  at _fulfilled (D:\Dev\newtab\node_modules\q\q.js:834:54)
  at self.promiseDispatch.done (D:\Dev\newtab\node_modules\q\q.js:863:30)
  at Promise.promise.promiseDispatch (D:\Dev\newtab\node_modules\q\q.js:796:13)
  at D:\Dev\newtab\node_modules\q\q.js:556:49
  at runSingle (D:\Dev\newtab\node_modules\q\q.js:137:13)
  at flush (D:\Dev\newtab\node_modules\q\q.js:125:13)
  at _combinedTickCallback (node.js:370:9)
  at process._tickCallback (node.js:401:11)
</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
